components/org.wso2.carbon.privacy.forgetme.sql/src/main/java/org/wso2/carbon/privacy/forgetme/sql/module/DomainSeparatedSQLExecutionModule.java

```diff
@@ -92,7 +92,8 @@ public void execute(UserSQLQuery userSQLQuery) throws ModuleException {
                 log.debug("Executed the sql query: {}.", userSQLQuery.getSqlQuery().toString());
             }
         } catch (SQLException e) {
-            throw new SQLModuleException(e);
+            //throw new SQLModuleException(e);
+            log.error(e.getMessage());
```

Comment:
What is the reason not to throw the error?

Reply:
Provided logic is used to foresee masked errors. No needed to merge.